packages/react-components/react-charts-preview/src/components/CommonComponents/CartesianChart.tsx

```diff
@@ -659,6 +673,96 @@ export const CartesianChart: React.FunctionComponent<IModifiedCartesianChartProp
   const xAxisTitleMaximumAllowedWidth = svgDimensions.width - margins.left! - margins.right! - startFromX!;
   const yAxisTitleMaximumAllowedHeight =
     svgDimensions.height - margins.bottom! - margins.top! - removalValueForTextTuncate! - titleMargin;
+  /**
+   * When screen resizes, along with screen, chart also auto adjusted.
+   * This method used to adjust height and width of the charts.
+   */
+  function _fitParentContainer(): void {
```

Comment:
why are _getData and _fitParentContainer showing as additions

Reply:
Will fix this

Reply:
Just changed function positioning in code, that is why